github-runner-manager/tests/integration/conftest.py

```diff
@@ -101,3 +101,17 @@ def app_fixture(
     process = start_app(config_file, [])
     yield process
     process.kill()
+
+
+@pytest.fixture(name="web_started_app", scope="function")
+def web_started_app_fixture(
```

Comment:
Maybe rename to `app_already_started` . Do we really need tests for both cases (app during start and already started) ?

Reply:
I think we do not need it.
It was due to how the waiting for flask was done. If I change the waiting for flask ready to be done by health checks instead of waiting for logs, this will not be needed.
I will be making this change.